web/src/services/loans.service.js

```diff
@@ -147,8 +147,9 @@ const getActiveLoan = () => {
 
 const getFundingLoans = () => {
   return instance
-    .get("/get_state/FUNDING/1")
+    .get("/get_state/FUNDING/1/")
```

Comment:
Why is User ID hardcoded?

Reply:
The 1 is a flag for the result to include user info on each loan

Reply:
okok